ProcessLib/HT/CreateHTProcess.cpp

```diff
@@ -115,6 +115,25 @@ std::unique_ptr<Process> createHTProcess(
     DBUG("Use \'%s\' as specific_heat_capacity_fluid parameter.",
          specific_heat_capacity_fluid.name.c_str());
 
+
+    // Parameter for the thermal conductivity of the solid (only one scalar per
+    // element, i.e., the isotropic case is handled at the moment)
+    auto& thermal_dispersivity_longitudinal = findParameter<double>(
```

Comment:
@wenqing 
Maybe I misunderstood you. As far as I understand the thermo dispersion tensor consists of a conductive and a dispersive part. The dispersive part depends on the darcy velocitiy and for this reason can not computed directly after parsing the input. The two new parameters introduced in the PR are used for the computation of the dispersive part.